### PR TITLE
[espmilighthub] Describe group 0 handling

### PR DESCRIPTION
**Pull request by DrRSatzteil**
Created a new PR since I messed up the old one...

This PR aims to add documentation for the handling of group 0 / ALL group.

### CODE REVIEW

**Comment by fwolter** [comment]
@Skinah Can you take a look?

@DrRSatzteil You need to sign-off your commits. See https://www.openhab.org/docs/developer/contributing.html#sign-your-work The necessary commands are listed when you click on "Details" at the DCO check below. You can also post the sign-off tag here in the comments.

**Comment by DrRSatzteil** [comment]
Signed-off-by: Thomas Lauterbach <thomas_lauterbach@arcor.de>

**Comment by DrRSatzteil** [comment]
Just like this? I only edited the README in the browser so I did not clone the repository locally. Last time I tried the signoff from cli I somehow completely messed up the PR XD

**Comment by fwolter** [comment]
That should be sufficient. Next time you could paste the sign-off message in the commit description whithin the web interface.

**Comment by DrRSatzteil** [comment]
It says "Verified" now so I think this worked... Thank you for your help!

File: bundles/org.openhab.binding.mqtt.espmilighthub/README.md
Comment (Skinah) [inline bundles/org.openhab.binding.mqtt.espmilighthub/README.md:160]:
```suggestion
While this functionality can also be achieved by using openHAB groups with even greater flexibility, the group 0 must be setup if you want to capture physical remote control events for the ALL group, and keep physical devices synchronized to their openHAB representations.
```

File: bundles/org.openhab.binding.mqtt.espmilighthub/README.md
Comment (Skinah) [inline bundles/org.openhab.binding.mqtt.espmilighthub/README.md:166]: Do not mention a version of openHAB, as when V4 is released this becomes outdated and hard to know it needs updating.

```suggestion
To create the group, use textual files or the openHAB UI to manually add a Thing with the correct Unique ID as described in section [Important for Textual Configuration](#important-for-textual-configuration).
```

File: bundles/org.openhab.binding.mqtt.espmilighthub/README.md
Comment (Skinah) [inline bundles/org.openhab.binding.mqtt.espmilighthub/README.md:167]:
```suggestion
To create a Thing for the group 0, simply create a new Thing that has the same type as one of the auto discovered Things of the same remote and modify the ThingUID as described in section linked above.
```

**Review by Skinah — CHANGES_REQUESTED** [review]
Thank you for the submission/contribution. If you accept any of the suggestions, please make sure your approval comment has the signoff as the last line.

**Comment by DrRSatzteil** [comment]
Signed-off-by: Thomas Lauterbach thomas_lauterbach@arcor.de

**Comment by DrRSatzteil** [comment]
Sorry I don't get it how to sign off these commits correctly... I added the sign-off virtually everywhere, any help would be appreciated.

**Comment by Skinah** [comment]
First why it has happened:
You have done 4 commits, the inital one and then 3 of my suggestions. The very first initial commit was missing the signoff completely. Then the three suggestions it appears you did try and my guess is you did not add the triangles around your email... 

Wont work
```
Signed-off-by: Thomas Lauterbach thomas_lauterbach@arcor.de
```

Should work if that name and email matches your github account.
```
Signed-off-by: Thomas Lauterbach <thomas_lauterbach@arcor.de>
```

It is easy to forget and it must be the very last line.

As to how to fix I am no expert and it can be done with git on the command line and how to do it is outlined by clicking on the details link next to where it tells you it failed...

https://github.com/openhab/openhab-addons/pull/10473/checks?check_run_id=2426281192

If you can not do it, just wait till the second person comes to review as sometimes it is waived for first time contributions. Hopefully the above makes it clear why it went wrong so next time it is a smooth process.

**Review by DrRSatzteil — COMMENTED** [review]
Signed-off-by: Thomas Lauterbach <thomas_lauterbach@arcor.de>

**Review by DrRSatzteil — COMMENTED** [review]
Signed-off-by: Thomas Lauterbach <lauterbachthomas@gmail.com>

File: bundles/org.openhab.binding.mqtt.espmilighthub/README.md
Comment (Skinah) [inline bundles/org.openhab.binding.mqtt.espmilighthub/README.md:157]:
```suggestion
## Using Group 0
```
Headings have each word capitalised. If you accept the change, be sure to add the signoff as the last line.

**Comment by DrRSatzteil** [comment]
> First why it has happened:
> You have done 4 commits, the inital one and then 3 of my suggestions. The very first initial commit was missing the signoff completely. Then the three suggestions it appears you did try and my guess is you did not add the triangles around your email...
> 
> Wont work
> 
> ```
> Signed-off-by: Thomas Lauterbach thomas_lauterbach@arcor.de
> ```
> 
> Should work if that name and email matches your github account.
> 
> ```
> Signed-off-by: Thomas Lauterbach <thomas_lauterbach@arcor.de>
> ```
> 
> It is easy to forget and it must be the very last line.
> 
> As to how to fix I am no expert and it can be done with git on the command line and how to do it is outlined by clicking on the details link next to where it tells you it failed...
> 
> https://github.com/openhab/openhab-addons/pull/10473/checks?check_run_id=2426281192
> 
> If you can not do it, just wait till the second person comes to review as sometimes it is waived for first time contributions. Hopefully the above makes it clear why it went wrong so next time it is a smooth process.

Thank you for your explanations. Since I messed up the first PR by trying to sign from the CLI (by following the steps described in the failing check) I think I will prefer the waiting option. However I noticed that I indeed used a different mail address in the comments than the one that I used for my GitHub account...

**Review by Skinah — APPROVED** [review]
All looks good here.

**Review by DrRSatzteil — COMMENTED** [review]
Signed-off-by: Thomas Lauterbach <2452988+DrRSatzteil@users.noreply.github.com>

**Review by fwolter — APPROVED** [review]

**Pull request merged**
